packages/ui/src/components/Button/index.tsx

```diff
@@ -1,41 +1,104 @@
+import { I18nKey } from '@logto/phrases-ui';
 import classNames from 'classnames';
-import { ReactNode } from 'react';
+import { HTMLProps, ReactElement, ReactNode, useEffect, useRef, useState } from 'react';
+import { useTranslation } from 'react-i18next';
 
+import DangerousRaw from '../../../../console/src/components/DangerousRaw';
+import { Ring as Spinner } from '../../../../console/src/components/Spinner';
 import * as styles from './index.module.scss';
 
-export type Props = {
+export type ButtonType =
+  | 'primary'
+  | 'secondary'
+  | 'danger'
+  | 'outline'
+  | 'plain'
+  | 'default'
+  | 'branding';
```

Comment:
UI does not have those different styles as console does. We may have a shared design system components in some public design package that can be shared by both UI and consoles. But better keep each package clean at this point.

Reply:
Sure, will update according to this.